components/net_traits/image_cache_thread.rs

```diff
@@ -37,6 +39,12 @@ pub enum ImageState {
     NotRequested,
 }
 
+impl From<bincode::serde::DeserializeError> for ImageState {
+    fn from(_: bincode::serde::DeserializeError) -> ImageState {
+        ImageState::LoadError
+    }
+}
+
```

Comment:
That's the error type from the Result that was previously unwrapped. It wouldn't compile without converting the error to ImageState.